lang/en/docs/cli/run.md

```diff
@@ -33,16 +33,24 @@ yarn run test
 Running this command will execute the script named `"test"` in your
 `package.json`.
 
-You can pass additional arguments to your script by using `--`.
+You can pass additional arguments to your script by passing them after the .
 
 ```sh
-yarn run test -- -o --watch
+yarn run test -o --watch
 ```
 
 Running this command will execute `jest -o --watch`.
 
 `[script]` can also be any locally installed executable that is inside `node_modules/.bin/`.
 
+It's also possible to leave out the `run` in this command, each script can be executed with its name: 
+
+```sh
+yarn test -o --watch
+```
+
+Running this command will do the same as `yarn run test -o --watch`
```

Comment:
Maybe add a reminder that it's only true for words that are not builtin commands, and that using the `run` syntax is preferable in scripts because of its added safety?